airlock/views/request.py

```diff
@@ -364,14 +365,18 @@ def request_release_files(request, request_id):
                     "response": err.response,
                     "type": err.response.headers["Content-Type"],
                 },
+                status=502,
```

Comment:
open to suggestions on which status code to use here, although I think I'm looking for  4xx or 5xx

Reply:
how about `status=err.status_code`? That should pass through the error from job-server API, which will be 4xx or 5xx (in order to have raised HTTPError?)

Reply:
yeah, fair enough, in my local dev env it returns `404` (as the workspace doesn't exist on prod job-server) and & in tests it returns `403`, that should work.